cli/p2p_replicator_delete.go

```diff
@@ -11,27 +11,36 @@
 package cli
 
 import (
+	"encoding/json"
+
 	"github.com/libp2p/go-libp2p/core/peer"
 	"github.com/spf13/cobra"
 
 	"github.com/sourcenetwork/defradb/client"
 )
 
 func MakeP2PReplicatorDeleteCommand() *cobra.Command {
+	var collections []string
 	var cmd = &cobra.Command{
-		Use:   "delete <peer>",
+		Use:   "delete [-c, --collection] <peer>",
 		Short: "Delete a replicator. It will stop synchronizing",
```

Comment:
question: Should this (and `Long`), not now read `Delete replicators...`? It looks like it can now delete multiple.

Reply:
If you are referring to deleting multiple collections replicated to a single peer you are correct.